src/main/include/subsystems/Climber.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020 FRC Team 3512. All Rights Reserved.
+
+#pragma once
+
+#include <frc/DoubleSolenoid.h>
+#include <frc/Joystick.h>
+#include <frc/PowerDistributionPanel.h>
+#include <frc/Solenoid.h>
+#include <rev/CANSparkMax.h>
+
+#include "Constants.hpp"
+#include "communications/PublishNode.hpp"
+#include "subsystems/SubsystemBase.hpp"
+
+
+/**
+ *  The climber composes of an winch, elevator, and traverser.
+ *
+ *  The elevator reachs for the climbing bar and then the winch grabs onto the
+ * bar. Then, if there are changes that are necessary, then the transverser is
```

Comment:
By changes, do you mean moving back and forth to adjust center of gravity on the bar?